packages/service/src/domain/commands/processors/stock-command.ts

```diff
@@ -0,0 +1,45 @@
+import fetch from 'node-fetch';
+import Command from '../model/command';
+import CommandDefinition from '../model/command-definition';
+import CommandProcessingResponse, { makeSingleTextProcessingResponse } from '../model/command-processing-response';
+
+async function stockCommandProcessor(command: Command): Promise<CommandProcessingResponse> {
+  if (command.rawArgs === '') {
+    throw new Error('Podaj nazwę spółki');
+  }
+  const res = await fetch(`https://www.bankier.pl/inwestowanie/profile/quote.html?symbol=${command.rawArgs}`);
+  const data = await res.text();
+  const divVithPrice = '<div class="profilLast">';
+  const indexOfDivVithPrice = data.indexOf(divVithPrice);
+
+  if (indexOfDivVithPrice < 0) {
+    throw new Error(`Nie znaleziono ceny dla spółki ${command.rawArgs}`);
+  }
+  const priceBeginningIndex = indexOfDivVithPrice + divVithPrice.length;
+  const priceLength = data.indexOf('</div>', priceBeginningIndex) - priceBeginningIndex;
+
+  const price = data.substr(priceBeginningIndex, priceLength);
```

Comment:
A może regexem by się to dało ogarnąć?

Reply:
Napisałem regexpa z grupami, sporo się uprościło, choć sam w sobie regexp nie jest prosty